### ISSUE TITLE
Use image instead of unicode for attachments icon column

### ISSUE THREAD

**Issue by OlivierMary** [keepassxreboot/keepassxc#1505]
Hi,
I just update/build keepassxc and new column needed/not needed ?

What is this column ?

![screenshot_20180221_213501](https://user-images.githubusercontent.com/4348675/36503989-53b6087a-174f-11e8-909f-01750ae197ad.png)

## Debug Info

KeePassXC - Version 2.3.0-snapshot
Build Type: Snapshot
Revision: 0237b21

Libraries:
- Qt 5.10.1
- libgcrypt 1.8.2

Operating system: Arch Linux
CPU architecture: x86_64
Kernel: linux 4.15.3-2-ARCH

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by phoerious** [comment]
The font you are using is missing some unicode glyphs. Try using a different font.

**Comment by OlivierMary** [comment]
Main system font ? I tried Noto sans, DejaVu Sans Nerd Font same problem.

**Comment by OlivierMary** [comment]
![screenshot_20180221_222235](https://user-images.githubusercontent.com/4348675/36506065-c3340f84-1755-11e8-88e9-72fef04de219.png)

**Comment by phoerious** [comment]
The font you are using for KeePassXC. The character in question is U+0001f4ce, which is a paper clip.

**Comment by OlivierMary** [comment]
@phoerious for me I can't choose any font in properties, that just take main font of my system.

**Comment by phoerious** [comment]
Can you see this character?  --->  📎 <---
And if yes, can you copy and use it in KeePassXC or other applications?

**Comment by OlivierMary** [comment]
📎 Yes I can edit too 📎

**Comment by phoerious** [comment]
Can you use it in KeePassXC? I realized that GitHub keeps replacing it with images, so copy it from the link above.

**Comment by OlivierMary** [comment]
I try in note same I have a rectangle. I try in other app

**Comment by phoerious** [comment]
Please try it in other Qt apps and if possible in a Gtk app for comparison.

**Comment by OlivierMary** [comment]
📎
![screenshot_20180221_223915](https://user-images.githubusercontent.com/4348675/36506855-19205aea-1758-11e8-9b80-6f6ae88a8ba1.png)

**Comment by phoerious** [comment]
Unfortunately, I don't have a solution for you, but the chances of this actually being a KeePassXC bug are very low. It's either a bug in your particular Qt version (also unlikely, I use the same, also on Arch) or a fontconfig/freetype/whatever misconfiguration on your system (most likely).

**Comment by OlivierMary** [comment]
I try many fonts + reboot, same problem. But Ok for column usage maybe we can put the name of column in column choice (right-click on table header). 
I just thought it was a translation problem, but no :)

**Comment by phoerious** [comment]
This string is not translatable.

**Comment by OlivierMary** [comment]
Ok I found: for me it's U+f0c6 >>   << bad in github :( but ok in keepass (note column):
![screenshot_20180221_230135](https://user-images.githubusercontent.com/4348675/36507842-30fb1850-175b-11e8-8555-dd85056331a0.png)

**Comment by phoerious** [comment]
Okay, something is clearly broken on your system: http://www.fileformat.info/info/unicode/char/f0c6/index.htm
Probably some weird encoding issue.

**Comment by OlivierMary** [comment]
nerdfont for U+f0c6

One of this pakage give me U+0001f4ce  in VSCode
```
Paquet (13)                Nouvelle version    Changement net  Taille du téléchargement

extra/xorg-fonts-alias     1.0.3-1                   0,02 MiB                          
extra/libunicodenames      1.2.2-1                   2,67 MiB                  0,37 MiB
extra/ttf-arphic-ukai      0.2.20080216.1-5         16,36 MiB                  7,56 MiB
extra/ttf-arphic-uming     0.2.20080216.1-5         20,08 MiB                  7,19 MiB
extra/ttf-freefont         20120503-4               10,30 MiB                  3,94 MiB
community/ttf-hanazono     20170904-2               51,03 MiB                 20,85 MiB
extra/ttf-junicode         1.001-1                   2,53 MiB                  1,01 MiB
extra/ttf-linux-libertine  5.3.0-4                  12,45 MiB                  3,54 MiB
extra/ttf-mph-2b-damase    001.000.4.dfsg.2-6        0,82 MiB                  0,36 MiB
community/ttf-symbola      10.23-1                   2,33 MiB                  0,99 MiB
extra/ttf-tibetan-machine  1.901-7                   4,30 MiB                  0,81 MiB
extra/ttf-ubraille         001.000-8                 0,15 MiB                  0,01 MiB
community/unicode-emoji    5.0-2                     0,61 MiB                  0,05 MiB

```
![screenshot_20180221_231218](https://user-images.githubusercontent.com/4348675/36508315-ad154d4c-175c-11e8-93f9-85bfee991c5b.png)
VSCode use Droid sans mono, monospace I try on main font for my system no change in keepass.

I give up, I can make test if needed, ask me.

**Comment by phoerious** [comment]
It seems you are not the only one with the problem. Although I still doubt that it's really a bug on our side, we need to find a solution.

**Comment by yan12125** [comment]
I had once the same issue. Installing noto-fonts-emoji and relogin fixed it.

**Comment by phoerious** [comment]
Maybe we need to use another character that's not inside the emoji range for better compatibility.

**Comment by OlivierMary** [comment]
Can't use an SVG ?

**Comment by droidmonkey** [comment]
I have implemented the paperclip using an image; PR will be posted

**Comment by phoerious** [comment]
Fixed in #1516

**Comment by OlivierMary** [comment]
![image](https://user-images.githubusercontent.com/4348675/36762174-a5716e7a-1c22-11e8-938c-7b7a216837b8.png)
@phoerious @droidmonkey  with the image it's black over black/grey (if black/dark OS UI) with SVG you can use text color no ? It's purely decorative of course.
Thanks @droidmonkey for the fast PR :+1:

**Comment by phoerious** [comment]
Good catch. Could you create new issue for that, please? We can fix that in 2.3.1.

**Comment by OlivierMary** [comment]
Okey
